src/System.ComponentModel.TypeConverter/src/System/ComponentModel/LicenseManager.cs

```diff
@@ -599,18 +564,9 @@ public CLRLicenseContext(LicenseUsageMode usageMode, Type type)
 
                 public override LicenseUsageMode UsageMode { get; }
 
+                public override string GetSavedLicenseKey(Type type, Assembly resourceAssembly) => (type == _type) ? _key : null;
 
-                public override string GetSavedLicenseKey(Type type, Assembly resourceAssembly)
-                {
-                    return type == _type ? _key : null;
-                }
-                public override void SetSavedLicenseKey(Type type, string key)
-                {
-                    if (type == _type)
-                    {
-                        _key = key;
-                    }
-                }
+                public override void SetSavedLicenseKey(Type type, string key) => _key = (type == _type) ? key : _key;
```

Comment:
Else it would not allow me to use ? : ternary operator... but I'm removing it as formatting changes unrelated to the PR are to be removed.